src/index.js

```diff
@@ -20,7 +21,7 @@ export default class Zeroconf extends EventEmitter {
   addDeviceListeners () {
 
     if (Object.keys(this._dListeners).length) {
-      return this.emit('error', new Error('RNZeroconf listeners already in place.'))
+      return this.emit('error', 'RNZeroconf listeners already in place.')
```

Comment:
Why emit a simple string?

Reply:
I only want to know if an error has ocurred (string is enough). If you pass an Error, you will get a red screen in debug mode, and an app crash in release mode

Reply:
fair

Reply:
This can't be right?
See my PR: https://github.com/balthazar/react-native-zeroconf/pull/79 (based on following issue: https://github.com/balthazar/react-native-zeroconf/issues/45)

The EventEmitter code expects an error object? See: https://github.com/nodejs/node/blob/908292cf1f551c614a733d858528ffb13fb3a524/lib/events.js#L187